neurons/config.py

```diff
@@ -124,13 +125,21 @@ def add_args(neuron_type: NeuronType, parser):
             action="store_true",
             help="Set this flag to disable setting the weights to network."
         )
+
         parser.add_argument(
             "--hf_results_path",
             action="store_true",
             help="Set this flag to select the location where you are want to store your hf_results data",
             default=os.path.join(Path(os.path.dirname(__file__)).parent, "hf_validation.parquet"),
         )
 
+        parser.add_argument(
+            "--db_storage_path",
+            type=str,
+            help="Set this flag to specify the file path in which your validator db information will be saved to.",
+            default="validator_storage.db"
```

Comment:
Usually in project we're using os.path.join(root_dir, 'file.sqlite') to avoid people who are running multiple projects on same machine the PYTHONPATH issue. 

The sqlite database name should be named in PascalCase ( i.e. **SomeDb**.sqlite). The extension should be **.sqlite**